code/modules/reagents/chemistry/reagents/unique/eigenstasium.dm

```diff
@@ -0,0 +1,110 @@
+////////////////////////////////////////////////////////////////////////////////////////////////////
+//										EIGENSTASIUM
+///////////////////////////////////////////////////////////////////////////////////////////////////
+//eigenstate Chem
+//Teleports you to the creation location and back
+//OD teleports you randomly around the Station and gives you a status effect
+//The status effect slowly send you on a wild ride and replaces you with an alternative reality version of yourself unless you consume eigenstasium/bluespace dust.
+//During the process you get really hungry, then some of your items slowly start teleport around you,
+//then alternative versions of yourself are brought in from a different universe and they yell at you.
+//and finally you yourself get teleported to an alternative universe, and character your playing is replaced with said alternative
+//Spraying this on lockers allows you to create eigenlinked lockers - see the eigenstate subsystem for using this to create your own links
+
+/datum/reagent/eigenstate
+	name = "Eigenstasium"
+	description = "A strange mixture formed from a controlled reaction of bluespace with plasma, that causes localised eigenstate fluxuations within the patient"
+	taste_description = "wiggly cosmic dust."
+	color = "#5020F4"
+	overdose_threshold = 15
```

Comment:
This is enough to fit in a syringe gun, is this acceptable?

Reply:
I'm on the fence about it honestly - we could try it and if it becomes a pain then up it to 20? I forgot that removing the addiction mechanics meant that the 16u requirement went back to 15u.